languages/python/README.md

```diff
@@ -33,8 +33,10 @@ file of this project for the most up-to-date information.
 ## Correctness and security ##
 
 Our **standard** code vulnerability scanners in cisagov are
-[LGTM](https://lgtm.com) and [Snyk](https://snyk.io).
-By **default** all repositories in the organization are scanned by these tools.
+[CodeQL](https://codeql.github.com/), [Snyk](https://snyk.io), and
+[Dependabot](https://docs.github.com/en/code-security/dependabot).  By
```

Comment:
This is not a blocking thought but do we want to sort these alphabetically?

Reply:
Please see commit f9d28c5.